### PR TITLE
5147: Added canned data for users

### PR DESCRIPTION
**Pull request by CallumBradburyBJSS**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.ViewModels.OrganisationUsers
+{
+    public sealed class GetAllOrganisationUsersViewModel
+    {
+        [JsonProperty("users")]
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs]: Do we need `JsonProperty` the attribute here?

File: src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/OrganisationUserViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        [JsonProperty("name")]
+        public string Name { get; set; }
+        
+        [JsonProperty("phoneNumber")]
+        public string PhoneNumber { get; set; }
+        
+        [JsonProperty("emailAddress")]
+        public string EmailAddress { get; set; }
+
+        [JsonProperty("isDisabled")]
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/OrganisationUserViewModel.cs]: Do we need `JsonProperty` the attributes here?

File: src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs (hunk 1 above)
Comment (CallumBradburyBJSS) [reply src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs]: No, I just like having it incase someone does a rename sometime in the future, and misses the json property. I think as we know for sure the contract we're implementing, we should probably implicitly state that is what we want.

Comment (bjss-chris) [reply src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs]: Yeah, it makes sense for renames – but it should be `JsonPropertyName` as we're currently using the `System.Text` serializer although I don't know if that's intentional or accidental (it's the default JSON serializer now).

Comment (CallumBradburyBJSS) [reply src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs]: I have removed the JsonProperty attributes for now, to get the Canned Data in, but will reintroduce them later

**Pull request merged**